python/ray/data/read_api.py

```diff
@@ -2282,6 +2283,9 @@ def from_tf(
 @PublicAPI
 def from_torch(
     dataset: "torch.utils.data.Dataset",
+    parallelism: int = -1,
```

Comment:
We also need to update this documentation: https://docs.ray.io/en/master/data/working-with-pytorch.html#migrating-from-pytorch-datasets-and-dataloaders